database/seeder/DatabaseSeeder.php

```diff
@@ -210,6 +217,12 @@ public function run()
             ]);
         };
 
+        //Demodata Seeder
+        if(is_module_installed('visiosoft.module.demodata'))
```

Comment:
good

Reply:
are you sure this will work if demodata is not installed? Or will it give a Class not found Error?

Reply:
`is_module_installed` and ı'm tested!